docs/intro.txt

```diff
@@ -80,16 +80,16 @@ The passes applied by different levels of optimisation are specified in the tabl
      - self.rebase_pass [2]
    * -
      - `ZZPhaseToRz <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.ZZPhaseToRz>`_
-     - `ZZPhaseToRz <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.ZZPhaseToRz>`_
-   * -
-     - `RemoveRedundancies <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.RemoveRedundancies>`_
      - `RemoveRedundancies <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.RemoveRedundancies>`_
    * -
-     - `auto_squash_pass [4] <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.auto_rebase.auto_squash_pass>`_
+     - `RemoveRedundancies <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.RemoveRedundancies>`_
```

Comment:
Looks like some duplication here. `RemoveRedundancies`, `auto_squash_pass` and `SimplifyInitial` are mentioned twice. Is this intended?

Reply:
Here is how the table appears in the built html. The source rst file is not very readable.
<img width="617" alt="Screenshot 2022-10-25 at 13 44 41" src="https://user-images.githubusercontent.com/93673602/197776410-e5d7d689-6333-4a81-839f-19d22f18d203.png">

Reply:
Ah OK, I see.